controls/radgridview/columns/column-groups.md

```diff
@@ -99,13 +91,67 @@ This mode is especially useful in more complicated scenarios where additional fe
 
 >important When the column group virtualization feature is enabled, __column group names should be unique__.
 
+## Add or Remove Multiple Column Groups at Once
+
+As of **R3 2020 SP** the ColumnGroups property is of type **RadObservableCollection** which exposes the **AddRange** and **RemoveRange** methods that have to be used when adding or removing multiple groups at once to avoid performance issues.

```

Comment:
Can we phrase this more positive - "that have to be used when adding or removing multiple groups at once to avoid performance issues."? 

For example: "As of **R3 2020 SP** the ColumnGroups property is of type **RadObservableCollection** which exposes the **AddRange** and **RemoveRange** methods that can be used when adding or removing multiple groups at once in order to improve performance. " - or something along those lines